setup.py

```diff
@@ -74,8 +74,8 @@
                                            libraries=libname,
                                            extra_link_args=lflags_arg)],
 	  scripts = ['rdiff-backup', 'rdiff-backup-statistics'],
-	  data_files = [('share/man/man1', ['rdiff-backup.1', 'rdiff-backup-statistics.1']),
+	  data_files = [('share/man/man1', ['docs/rdiff-backup.1', 'docs/rdiff-backup-statistics.1']),
```

Comment:
As you'll need anyway to do some changes based on my comments, I'll let you rebase your setup.py after I've merged my changes. Sorry, first come first served :stuck_out_tongue:

Reply:
Eric, sure not a problem.